### ISSUE TITLE
Select folders and select all folders is not allowed in storage

### ISSUE THREAD

**Issue by a0m0rajab** [supabase/supabase#29905]
# Bug report

- [ ] I confirm this is a bug with Supabase, not with my own application. 
- [ ] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

I can not select any folder or select all folders in storage, here is a similar issue: https://github.com/supabase/supabase/issues/10510

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by Hallidayo** [comment]
Hi @a0m0rajab - thank you for opening the issue, Do you have a screenshot/video of this issue or is the issue no longer happening for you?

**Comment by a0m0rajab** [comment]
yeah, it's still happening, when you go to storage, you can select items but not folders, this is the issue for me.

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
